src/components/ability/index.js

```diff
@@ -4,12 +4,15 @@ import Button from '../button';
 
 import './styles.scss';
 
-const Ability = ({ name, value, increment, decrement }) => {
+const Ability = ({ name, value, minValue, increment, decrement }) => {
     return (
         <>
             <div className="ability-score__container">
                 <span className="ability-score__text">{name}:</span>
-                <div className="ability-score__button ">
+                <div
+                    className="ability-score__button"
+                    style={{ visibility: value === 20 ? 'hidden' : 'visible' }}
```

Comment:
For any future feature that may increase the maximum beyond 20, I think it would be good to have a `maxValue` here instead of a hardcoded value.

Reply:
The max value already exists as a constant (MAX_ABILITY_SCORE).

Reply:
While that does exist, the implementation I'm suggesting would allow the maximum ability score value to be changed at runtime. Perhaps it should be part of a separate PR, in which case line 14 should be changed to use `MAX_ABILITY_SCORE`.

Reply:
Why would the max ability score need to be changed at runtime? 
I think that the 20 should probably be changed to `MAX_ABILITY_SCORE`